week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
+            <div class="text">버튼을 눌러 시작하세요!</div>
+        </div>
+        
+        <div class="middle">
+            <div id="human_select"><img id="img" src="./img/paper.png"/></div>
+            <div id="robot_select"><img id="robotimg" src="./img/paper.png"/></div>
+        </div>
+        <div class="bottom">
+            <button onclick="scissor_button()" id="img_toScissor">가위</button>
+            <button onclick="rock_button()" id="img_toRrock">바위</button>
+            <button onclick="paper_button()" id="img_toPaper">보</button>
+        </div> 
+        <button id="reset">초기화</button>
+    </div>
+
+    <script>
+        let humanScore  = document.getElementById("humanscore");
+        let computerScore = document.getElementById("computerscore");
+        const resetbutton = document.getElementById("reset");
+        const text = document.getElementsByClassName("text");
+        let human = 0;
+        let computer = 0;
+        resetbutton.onclick = ()=>{
+            computerScore.innerText = "0";
+            humanScore.innerText = "0";
+            //승리 출력
+            const text = document.getElementsByClassName("text");
+                for (let i = 0; i < text.length; i++) {
+                    text[i].innerText = "버튼을 눌러 시작하세요!";
+                }
```

Comment:
getElement"**s**"ByClassName();은 컬렉션을 반환합니다. 클래스 요소 한 개만 불러오고 싶다면 querySelector()를 사용합니다. 컬렉션이 아닌 한 개의 요소는 반복문을 사용할 필요가 없습니다.
```suggestion
                text.innerText = "버튼을 눌러 시작하세요!";
```